src/components/containers/ContainersListContent.js

```diff
@@ -72,17 +71,10 @@ const ContainersListContent = ({
     withToken(token, api.containers.listExport)().then(res => res.data)
 
   return <>
-    <AppPageHeader
-      title="Containers"
-      extra={[
-        <ExportButton
-          exportFunction={listExport}
-          fileName={'containers'}
-        />
-        ,
-        ...actionsToButtonList("/containers", actions)
-      ]}
-    />
+    <AppPageHeader title="Containers" extra={[
+      <ExportButton exportFunction={listExport} filename={'containers'}/>,
+      ...actionsToButtonList("/containers", actions)
+    ]}/>
```

Comment:
You have missed this one: `filename='containers'`, not `filename={'containers'}`

Reply:
No, actually it's `filename="containers"`, this project uses double-quotes for all strings. I personally prefer single-quotes usually but consistence is more important than preference.